email-ballerina/Ballerina.lock

```diff
@@ -0,0 +1,4 @@
+org_name = "ballerina"
```

Comment:
We don't need to commit this file ryt? This will anyway be removed with gradle clean.

Reply:
Removed.